superset-frontend/src/dashboard/components/SliceAdder.jsx

```diff
@@ -221,7 +222,7 @@ class SliceAdder extends React.Component {
             innerRef={dragSourceRef}
             style={style}
             sliceName={cellData.slice_name}
-            lastModified={cellData.changed_on_humanized}
+            lastModified={moment.utc(cellData.changed_on).fromNow()}
```

Comment:
This should not be changed (@dpgaspar can comment in more detail, but IIRC, the `changed_on` property needs to be constructed in the backend right now)

Reply:
changed_on_humanized have any translation. I will be glade to keep 'changed_on_humanized' but what I can change to have transaltion on it

I move the change on this PR : https://github.com/apache/superset/pull/20082
we can discuss about that on the new PR